README.md

```diff
@@ -1,4 +1,81 @@
-# StudentAPI-nodeJs
-* A restful API that perform CRUD operations on Students' and courses' data
-* student can take or drop courses
-* Link: https://danielstudent-api.herokuapp.com/api
+# Student Web Server
+
+Student Web Server is a RESTful API that manages students' and courses' information. The API performs CRUD operations on student and course data (stored in MongoDB), and returns the results in JSON format. The API can allow students to register for courses, and students can view their current courses.
+
+## Table of Contents
+
+- [Motivation](#motivation)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Testing](#api-testing)
+- [Tools](#tools)
+- [Heroku](#heroku)
+- [Demo](#demo)
+- [Authors](#authors)
+
+# Motivation
+
+Student Web Server was inspired by my college schedule builer. I wanted to create a web application that would allow students to create their own schedules by adding courses to their schedule or removing courses from their schedule.
```

Comment:
Careful of typos and poor grammar/spelling! These are negative signals to potential hiring managers
```suggestion
Student Web Server was inspired by my college schedule builder. I wanted to create a web application that would allow students to create their own schedules by adding courses to their schedule or removing courses from their schedule.
```